crates/matrix-sdk/src/room/timeline/read_receipts.rs

```diff
@@ -0,0 +1,228 @@
+// Copyright 2023 The Matrix.org Foundation C.I.C.
```

Comment:
I think you should put your own name here ^^

Reply:
Is that mandatory? I don't really care about it. That's why I always put the Matrix foundation, it'll always be true at some point.

Reply:
Hm, I guess probably not? Have asked.

Reply:
It seems like we really want you to put in your own name 😄